src/lib/time.js

```diff
@@ -1,7 +1,7 @@
 export function formatSecs(secs) {
   secs = Math.round(secs);
   if (secs < 60) {
-    return `${secs} s`;
```

Comment:
The version with a space is actually correct, see https://www.bipm.org/en/publications/si-brochure/section5-3.html. The other one might look better though.